bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideGroup.java

```diff
@@ -369,7 +364,7 @@ public boolean containsTarget(SPTarget target) {
 
     @Override
     public GuideGroup removeTarget(final SPTarget target) {
-        final ImList<GuideProbeTargets> updated = guideTargets.map(gpt -> gpt.removeTargetSelectPrimary(target));
```

Comment:
They actually do the exact same thing. It's just that when we divorced `GuideProbeTargets` from `OptionsList`, I decided to rename the method to make it more explicit that if it was the primary target being removed, then another primary was selected. You can see this in `OptionsList.UpdateOps.remove`, which is what is called by `GuideProbeTargets.remove`.